### DISCUSSION TITLE
Epub reader to highlight text (to serve as bookmarks) or just bookmark pages

### DISCUSSION THREAD

**Discussion by BaccanoMob** [gotson/komga#1515] [Ideas]
### Describe your suggested feature

This would be great feature to include to save interesting/informative section of the book (rather than trying to find the page manually in the web-ui). 

Currently the web reader supports read-progress which means it can track current page (even when scroll view is used). So when a page is bookmarked you can store a key value pair of bookmark title and page according to the progress. 

Highlighting the text should be similar though I am not sure how doable it is.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by BenTable** [comment]
It's been close to a year, but I'm commenting for support. A lot of times books'll have a glossary or a map that I'll want to look back on while reading, and it'd be alot more convenient to do so if a bookmark feature was added.

**Comment by mokomonsen** [comment]
This is the main thing I'm missing, and forcing me to use other apps on my devices, as I often highligth text og bookmark pages for later review.

**Comment by dtsdwarak** [comment]
+1 for the feature.

Would like for this to be available for PDF files too
